.github/workflows/realm-dart-linux.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
       - name: Run tests
         run: |
-          dart test -r expanded --test-randomize-ordering-seed random
+          dart test -r expanded -j 1 --test-randomize-ordering-seed random
```

Comment:
Why do we need this? It prevents parallel execution of tests. Is it not possible that will catch some corner cases, even if it does introduce some non-determinism?

Reply:
It is called out in the PR description. 
Our CI does not do parallel execution anyways it seems, I guess since it has a single CPU core per job.